marketplacetests/marketplace/regions/search_results.py

```diff
@@ -25,6 +23,12 @@ def search_results(self):
         search_results = self.marionette.find_elements(*self._search_result_locator)
         return [Result(self.marionette, result) for result in search_results]
 
+    def install_searched_app(self, search_term):
```

Comment:
If that is a case we need covered, it should have its own test. The purpose of this test if to test in-app payments, not whether we can install directly from search results, so I think we should remove it from here and check with @krupa to see if we need another test for installing directly from search results.